hashintel/hash
# Add example-graph.json to table block

## 🌟 What is the purpose of this PR?



This PR updates the table block to include an `example-graph.json` file for storing dummy entities, types, links the block might use. Previously the table block relied on dummy data provided by `mock-block-dock`. The `example-graph.json` ensures users can add dummy data blocks can use for demo purposes.
It also patches `block-scripts` to bundle `example-graph.json` when building blocks.

## 🔗 Related links





- [Add example-graph.json to spec and use in Block Hub](https://app.asana.com/0/home/1200704678189494/1202202445930679) _(internal)_


## 🔍 What does this change?




- Patches `block-scripts` to bundle `example-graph.json` file when building blocks
- Adds an `example-graph.json` file to the table block containing dummy entities/entity types useful for demo
- Removes legacy `__linkedData` in `table` example

## ⚠️ Known issues



## 🐾 Next steps



- Merge in https://github.com/blockprotocol/blockprotocol/pull/328 which includes code that displays the example graph tab in BP hub
- Update the repo url of the `table` block once https://github.com/blockprotocol/blockprotocol/pull/328 is merged
- Update block-scripts once the version has been updated


## ❓ How to test this?



Locally
1. Update any of the entities in `table/example-graph.json`
1. Confirm the update shows up when running `dev` in the table block

BP
1.  Check this deployment on BP => https://blockprotocol-rlnn60bqp-hashintel.vercel.app/@hash/blocks/table
1.  Confirm the Example Graph tab is present with data.


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202202445930679
